src/Grpc/versions.txt

```diff
@@ -1,2 +1,2 @@
-# The following files were downloaded from branch main at 2025-09-17 01:45:58 UTC
-https://raw.githubusercontent.com/microsoft/durabletask-protobuf/f5745e0d83f608d77871c1894d9260ceaae08967/protos/orchestrator_service.proto
+# The following files were downloaded from branch nytian/failure-details at 2025-10-01 21:51:24 UTC

```

Comment:
Reminder to update this to `main` once the protobuf changes in durabletask-protobuf are merged.